authentication/account/service/user_blackbox_test.go

```diff
@@ -177,8 +185,10 @@ func (s *userServiceBlackboxTestSuite) TestListUsersToNotifyBeforeDeactivation()
 		config.GetUserDeactivationInactivityNotificationPeriodDaysFunc = func() time.Duration {
 			return 30 * 24 * time.Hour // 30 days
 		}
+		var msgToSend []notification.Message
 		notificationServiceMock := servicemock.NewNotificationServiceMock(s.T())
 		notificationServiceMock.SendMessageAsyncFunc = func(ctx context.Context, msg notification.Message, options ...rest.HTTPClientOption) (r chan error, r1 error) {
+			msgToSend = append(msgToSend, msg)
```

Comment:
cI see the possibility that the msg come out of order.  As I understood, this is the [code](https://github.com/fabric8-services/fabric8-auth/blob/master/authentication/account/service/user.go#L147) which will call this `SendMessageAsyncFunc` and things run parallel due to this [code](https://github.com/fabric8-services/fabric8-auth/blob/master/authentication/account/service/user.go#L139)

Note: I am not totally sure about above thing.  I am adding this comment so that you can also think if that is possible or not.

Reply:
done in 4893646